lib/panoptes_client.rb

```diff
@@ -0,0 +1,43 @@
+require 'authenticator'
+require 'panoptes/client'
+
+class PanoptesClient
+  attr_accessor :client
+
+  def initialize(token)
+    @token ||= token
+    client
```

Comment:
You mean include a `client` instance in the initialize method signature here instead of building it via a PanoptesClient instance method? Happy to take a look at a specific suggestion and a case for why it's ideal, please open an issue.